x-pack/legacy/plugins/apm/e2e/run-e2e.sh

```diff
@@ -111,8 +113,7 @@ fi
 ##################################################
 echo "\n${bold}Waiting for Kibana to start...${normal}"
 echo "Note: you need to start Kibana manually. Find the instructions at the top."
-yarn wait-on -i 500 -w 500 http://localhost:$KIBANA_PORT > /dev/null
-
+yarn wait-on -i 500 -w 500 http://localhost:${KIBANA_PORT}/status > /dev/null
 echo "\n✅ Setup completed successfully. Running tests...\n"
 
 #
```

Comment:
Could we invoke here the rerun script? Here and there they are both using
```shell
yarn cypress run --config pageLoadTimeout=100000,watchForFileChanges=true
```

Reply:
We have discussed this in the IRC and here are the notes for future references.

I initially though to do so, but the `rerun` is just a workaround for the requirement of running the e2e tests in the CI using the same steps that the developers do locally.

Unfortunately the  steps that the developer do locally seems to have a pre-requirement of running the Kibana instance in advance and therefore it's up and running. See https://github.com/elastic/kibana/issues/61942 for further context.

So to avoid messing the `x-pack/legacy/plugins/apm/e2e/run-e2e.sh` with conditions such as if running in the CI then apply this particular condition and so on , I moved the logic to a script under the `ci` folder to be more specific about the responsibilities. 

As soon as the steps to run the e2e support running from scratch in a vanilla workspace then we will be able to get rid of the workaround until then I'm afraid this is the only way I've managed to implement it so far.